packages/boxel-ui/addon/src/icons.gts

```diff
@@ -20,6 +20,7 @@ import DropdownArrowFilled from './icons/dropdown-arrow-filled.gts';
 import DropdownArrowUp from './icons/dropdown-arrow-up.gts';
 import Exclamation from './icons/exclamation.gts';
 import ExclamationCircle from './icons/exclamation-circle.gts';
+import ExclamationTriangleFill from './icons/exclamation-triangle-fill.gts';
```

Comment:
There's a very similar icon `warning.svg` already existing here. Does that icon work?